README.md

```diff
@@ -43,6 +43,7 @@ If you want to contribute, please read [CONTRIBUTING.md](./CONTRIBUTING.md).
 * [ChatOps For Dummies](https://victorops.com/chatops-for-dummies/) – A high-level guide to understanding the origins of the practice, the benefits and tools needed to get started.
 * [ChatOps – Managing Operations in Group Chat](https://victorops.com/knowledge-drop/devops-docs/chatops-managing-operations-group-chat/) – The key ChatOps concepts and technologies by Jason Hand.
```

Comment:
@StanislavGrinkov thanks for contributing!

Seems like it's the same book by Jason Hand on [victorops.com](https://victorops.com/knowledge-drop/devops-docs/chatops-managing-operations-group-chat/) website. I think we can leave the old URL then :)

Reply:
Oh. Sorry for not checking. :) I thought this is different books because of different tiles.